modules/ROOT/partials/configuration/insertdatetime_element.adoc

```diff
@@ -5,6 +5,8 @@ When this option is enabled HTML5 time elements gets generated when you insert d
 
 Type: `+Boolean+`
 
+Default value : `+false+`
```

Comment:
```suggestion
Default Value : `+false+`
```
In most other places this is title case... we probably need to look at making them all consistent though so might be worth logging a Jira to do that.

Reply:
Created a Jira: DOC-1607